### ISSUE TITLE
Create better homepage

### ISSUE THREAD

**Issue by janpanschab** [actum/gulp-dev-stack#86]
Better `index.nunj` with basic bootstrap usage, reasonable text and HTML5 semantic elements.
Create also better structure in `tpl` folder. Mainly remove `partials` folder.

**Comment by vbulant** [comment]
Related: https://github.com/actum/gulp-dev-stack/issues/146

**Comment by vbulant** [comment]
Is the devstack the right place for Bootstrap examples or tutorials of any sort?
Shouldn’t it be the minimal boilerplate instead?

Another option is to create smaller projects focused on those topics (how to write/use x/y) so that anytime we want someone to learn something we hand them simple learning project and at the same time we keep the devstack simple.

**Comment by janpanschab** [comment]
Would be so nice.

**Comment by kettanaito** [comment]
@vbulant This may be irksome to maintain. Apart from technologies change (i.e. Bootstrap 4 grid uses different syntax than Bootstrap 3), each project may use its own versions of the same technology, or even a brand new one.

I like that homepage displays the list of available pages. Would be nice to display the same for available components. This brings in the question of creating a boilerplate for the devstack.

**Comment by vbulant** [comment]
@kettanaito 

> I like that homepage displays the list of available pages. Would be nice to display the same for available components.

I see, but that’s related to https://github.com/actum/gulp-dev-stack/issues/146 so let’s continue on this topic there.

**Comment by vbulant** [comment]
I’ve added **on hold** label coz I think we should discuss https://github.com/actum/gulp-dev-stack/issues/163 first. In case we go for it, this issue is not relevant anymore or at least it should be implemented after #163.

**Comment by DavidSlavik** [comment]
Closing because it's obsolete...
